editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -167,19 +167,36 @@ function Question(props: ComponentProps) {
 }
 
 function FindProperty(props: ComponentProps) {
-  const { postcode, single_line_address, town } = props.passport.data?._address;
-  return (
-    <>
-      <dt>Property</dt>
-      <dd>
-        {`${single_line_address.split(`, ${town}`)[0]}`}
-        <br />
-        {town}
-        <br />
-        {postcode}
-      </dd>
-    </>
-  );
+  const { source } = props.passport.data?._address;
+
+  if (source === "os") {
+    const { postcode, single_line_address, town } =
+      props.passport.data?._address;
+    return (
+      <>
+        <dt>Property</dt>
+        <dd>
+          {`${single_line_address.split(`, ${town}`)[0]}`}
+          <br />
+          {town}
+          <br />
+          {postcode}
+        </dd>
+      </>
+    );
+  } else {
+    const { x, y, title } = props.passport.data?._address;
+    return (
+      <>
+        <dt>Proposed address</dt>
+        <dd>
+          {`${title}`}
+          <br />
+          {`${x.toFixed(5)} Easting (X), ${y.toFixed(5)} Northing (Y)`}
```

Comment:
Eastings and Northings are generally presented as whole numbers - I think that's probably what the OS API would return? I think this would apply on the map page also.

Reply:
you're right! i think this actually highlights a projection bug i need to look at in the map repo, because my whole numbers being returned by the geojson event look like X -171, Y 70 for London-ish, whereas I'd expect positive, six-digit whole numbers for Easting/Northing, right?

Reply:
That's correct - whole numbers, positive for eastings / northings. You could cross reference something like https://gridreferencefinder.com/os.php to sense check. 

Let me know if you want another pair of eyes on this one 👍

Reply:
projection fix queued up over here! https://github.com/theopensystemslab/map/pull/261